src/net_processing.cpp

```diff
@@ -1015,7 +1018,7 @@ void PeerManagerImpl::ReattemptInitialBroadcast(CScheduler& scheduler)
 
         if (tx != nullptr) {
             LOCK(cs_main);
-            RelayTransaction(txid, tx->GetWitnessHash());
```

Comment:
9012512a57283bdcd0cc0562680d5de2f304af9f

Remove the above lock instead?

Reply:
A future PR is going to replace `cs_main` in `_RelayTransaction()` with a new internal net_processing lock, and we'll need to call the internal `_RelayTransaction()` function here. It makes sense to keep this as it is for now.

Reply:
I wasn't aware of that, makes sense then (you can resolve this thread).